portfolio/src/main/webapp/index.html

```diff
@@ -1,30 +1,34 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Zoe's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Zoe's Portfolio</h1>
-      <p>Welcome! This is my portfolio.</p>
-      <br>
-      <h2>A bit about me</h2>
-      <ul>STEP intern in summer 2020, on Youtube TPS team</ul>
-      <ul>I played around with pipettes and chemicals in a wet lab in summer 2019</ul>
-      <ul>Major in robotics🤖</ul>
-      <ul>Guilty pleasure: binge-watch The Big Bang Theory</ul>
-      <ul>Fun fact: ❤raw carrots ⛔cooked carrots</ul>
-      <br>
-      <h2>Where am I?</h2>
-      <button onclick="findZoe()">Find Zoe</button>
-      <div id="link-container">
-        <a id="link"></a>
-      </div>
-      <br>
-      <h2>Help me <button onclick="grow()"><b>Grow</b></button></h2>
-      <img id="basil" />
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Zoe's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div id="content">
+    <h1>Zoe's Portfolio</h1>
+    <p>Welcome! This is my portfolio.</p>
+    <br>
+    <h2>A bit about me</h2>
+    <ul>STEP intern in summer 2020, on Youtube TPS team</ul>
+    <ul>I played around with pipettes and chemicals in a wet lab in summer 2019</ul>
+    <ul>Major in robotics🤖</ul>
+    <ul>Guilty pleasure: binge-watch The Big Bang Theory</ul>
+    <ul>Fun fact: ❤raw carrots ⛔cooked carrots</ul>
+    <br>
+    <h2>Where am I?</h2>
+    <button onclick="findZoe()">Find Zoe</button>
+    <div id="link-container">
+      <a id="link"></a>
+    </div>
+    <br>
+    <h2>Help me <button onclick="grow()"><b>Grow</b></button></h2>
+    <img id="basil" />
+    <p id="endMsg"></p>
```

Comment:
This is the only line I changed the content. The other lines are touched because of the indentation.